### PR TITLE
update builders (component and test files)

### PR DESCRIPTION
**Pull request by chrisgalvan**
This PR updates the ` scripts/createComponent/builders.js` file to adhere to new standards.

- Test file references react-testing-library instead of enzime #850 
- Component file uses functional component style instead of classes. #845

### CODE REVIEW

**Comment by kylemh** [comment]
🤖 **This is a bot** 🤖<br /><br />🎉 Deployed Storybook preview! 🎉<br /><br />Click the link at the bottom of this comment to see it live.

Built with commit dc16c820139384f18ad9d57e59b810629b0f37d5

https://deploy-preview-884--operation-code-storybook.netlify.com

File: scripts/createComponent/builders.js [hunk 1]
```diff
@@ -3,30 +3,27 @@
...
-      string,
-    ]).isRequired,
-    className: string,
-  };
-
-  static defaultProps = {
-    className: undefined,
-  };
+${componentName}.propTypes = {
+  children: oneOfType([
```
Comment (kylemh) [inline scripts/createComponent/builders.js]: I think just `node` works fine! Also, can you require this?

File: scripts/createComponent/builders.js [hunk 2]
```diff
@@ -57,22 +49,16 @@ storiesOf('${componentName}', module)
...
 
-  it('should render with many props assigned', () => {
-    createSnapshotTest(
-      <${componentName} className="test-class">
-        Test
-      </${componentName}>,
-    );
-  });
+    expect(container.firstChild).toBeNull();
+  })
```
Comment (kylemh) [inline scripts/createComponent/builders.js]:
```javascript
it('should render', () => {
  const component = render(<${componentName} />);
  
  expect(component.container.firstChild).not.toBeNull();
});
```

Comment (chrisgalvan) [reply scripts/createComponent/builders.js]: great catch

**Pull request merged**